packages/core/src/lib/toast/toast-banner.svelte

```diff
@@ -20,7 +20,7 @@ export { extraClasses as cx };
 
 <div
   class={cx(
-    'relative flex w-auto items-center gap-2 border bg-medium p-2 text-sm',
+    'relative flex w-auto items-center gap-2 border bg-medium p-1 text-sm',
     extraClasses
   )}
 >
```

Comment:
Can't comment on a few things below because of GH, but a few problems I see:

- Where is the `w-[18px]` wrapping the `<Icon>`coming from?
    - According to Figma, the icon width is supposed to be 20px, the default for the `lg` variant
    - The `<div>` wrapper with a width override seems unnecessary
- The Figma says the message should have a margin-right of `16px`, so `mr-4` instead of `mr-auto`

Reply:
The `w-[18px]` was a carry-over from another state that has been totally discarded and I forgot to remove. Removing it in my next push